analysis_module/views/analysis_module.py

```diff
@@ -8,17 +8,18 @@
 from rest_framework.request import Request
 from rest_framework.decorators import api_view, permission_classes
 from rest_framework.permissions import IsAuthenticated
-from .serializers import (
+from analysis_module.serializers import (
     TopicModelDeepRequest,
     DeepEntriesSerializer,
     NgramsDeepRequest,
```

Comment:
These requests are not only coming from DEEP, it could be any external system. So, maybe need to rename it. Let me know if I understood differently.

Reply:
Makes sense. Done.